core/arch/arm/arm.mk

```diff
@@ -1,3 +1,15 @@
+# Setup compiler for this sub module
```

Comment:
"this sub module" refers to the submodule `core` (`sm=core`), in this new context it looks a bit cryptic.

Reply:
Fixed